packages/aws-cdk-lib/aws-synthetics/lib/canary.ts

```diff
@@ -684,7 +689,7 @@ export class Canary extends cdk.Resource implements ec2.IConnectable {
 
     // Only check runtime family is Node.js because versions prior to `syn-nodejs-puppeteer-3.3` are deprecated and can no longer be configured.
     if (!isNodeRuntime && props.artifactS3EncryptionMode) {
-      throw new Error(`Artifact encryption is only supported for canaries that use Synthetics runtime version \`syn-nodejs-puppeteer-3.3\` or later, got \`${props.runtime.name}\`.`);
```

Comment:
Would you be able to add a test for Playwright in `integ.canary-artifact-s3-encryption` to confirm this works as expected? Thanks

Reply:
@nmussy I'm sorry for my late response😓 I've just updated integ test and snapshots!